### PR TITLE
Add Samsung Galaxy S8+ (Exynos) (dream2lte)

### PR DESCRIPTION
**Pull request by vanyasem**
Can already produce a working build in it's current state

### CODE REVIEW

File: manifests/samsung_dream2lte.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <project path="device/samsung/dream2lte" name="universal8895/android_device_samsung_dream2lte" remote="hal" revision="cm-14.1" />
+  <project path="device/samsung/universal8895-common" name="vanyasem/android_device_samsung_universal8895-common" remote="hal" />
+  <project path="hardware/samsung" name="vanyasem/android_hardware_samsung" remote="hal" />
```
Comment (lnjX) [inline manifests/samsung_dream2lte.xml]: Please try out the `Halium/android_hardware_samsung` repository

---

Comment (lnjX) [reply manifests/samsung_dream2lte.xml]: ^^  (not outdated)

File: manifests/samsung_dream2lte.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="universal8895" fetch="https://github.com/universal8895" revision="cm-14.1" />
+
+  <project path="device/samsung/dream2lte" name="android_device_samsung_dream2lte" remote="universal8895" />
+  <project path="device/samsung/universal8895-common" name="vanyasem/android_device_samsung_universal8895-common" remote="hal" />
+  <project path="hardware/samsung" name="vanyasem/android_hardware_samsung" remote="hal" />
```
Comment (lnjX) [inline manifests/samsung_dream2lte.xml:7]: Please try out the `Halium/android_hardware_samsung` repository

File: manifests/samsung_dream2lte.xml (hunk 1 above)
Comment (vanyasem) [reply manifests/samsung_dream2lte.xml]: It's missing [7b26c18](https://github.com/universal8895/android_hardware_samsung/commit/7b26c188b7931f99a58fd0afcee3b9531495d13a). My device isn't officially supported by LineageOS, and thus device-specific commits aren't in the repo

File: manifests/samsung_dream2lte.xml (hunk 2 above)
Comment (vanyasem) [reply manifests/samsung_dream2lte.xml:7]: It's missing [7b26c18](https://github.com/universal8895/android_hardware_samsung/commit/7b26c188b7931f99a58fd0afcee3b9531495d13a). My device isn't officially supported by LineageOS, and thus device-specific commits aren't in the repo

Comment (lnjX) [reply manifests/samsung_dream2lte.xml:7]: Ok

**Pull request merged**